File: app/src/main/java/com/singh/gurtek/kotlinconductor/models/Photos.java
package com.singh.gurtek.kotlinconductor.models;

/**
 * Created by JacksonGenerator on 12/13/17.
 */

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;


public class Photos {
    @JsonProperty("perpage")
    private Integer perpage;
    @JsonProperty("total")
    private String total;
    @JsonProperty("pages")
    private String pages;
    @JsonProperty("photo")
    private List<PhotoItem> photo;
    @JsonProperty("page")
    private Integer page;
}

File: app/src/main/java/com/singh/gurtek/kotlinconductor/models/Photos.kt
package com.singh.gurtek.kotlinconductor.models

/**
 * Created by Gurtek singh
 * on on 12/13/17
 * gurtek@protonmail.com
 */

import com.fasterxml.jackson.annotation.JsonProperty


data class Photos(
        @JsonProperty("perpage")
        private val perpage: Int? = null,
        @JsonProperty("total")
        private val total: String? = null,
        @JsonProperty("pages")
        private val pages: String? = null,
        @JsonProperty("photo")
        private val photo: List<PhotoItem>? = null,
        @JsonProperty("page")
        private val page: Int? = null
)